packages/html/ds/src/header/components/headerMenu.html

```diff
@@ -38,11 +41,11 @@
         <span class="gi-text-sm">{{ link.label }}</span>
       </a>
     </li>
-    {% endfor %}
-    {% if props.searchUrl %}
-      <li class="xs:gi-hidden gi-mt-8">
-        {{ HeaderSearchContainer({"searchUrl": props.searchUrl, "dataElement": "search-container-mobile" , "dataTestId": "search-container-mobile"}) }}
-      </li>
+    {% endfor %} {% if props.searchUrl %}
```

Comment:
question: Is this formatting changes? If yes, then can we please revert it back to the original one as its more readable 😄